apps/analytics/views.py

```diff
@@ -211,3 +207,58 @@ def get_last_submission_datetime_analysis(request, challenge_pk, challenge_phase
     except:
         response_data = {'error': 'Bad request. Please try again later!'}
         return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+
+@throttle_classes([UserRateThrottle])
+@api_view(['GET', ])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def get_challenge_phase_submission_analysis(request, challenge_pk, challenge_phase_pk):
+    """
+    API to fetch
+    1. The total submissions count in a challenge phase.
+    2. The participant team count in a challenge phase.
+    3. Submission count for Submitted/Running/Failed/Cancelled/Finished/Submitting submissions.
+    """
+
+    challenge = get_challenge_model(challenge_pk)
+
+    challenge_phase = get_challenge_phase_model(challenge_phase_pk)
+
+    # Get the total submissions in a challenge phase
+    submissions = Submission.objects.filter(
+        challenge_phase=challenge_phase, challenge_phase__challenge=challenge)
+    submission_count = submissions.count()
+
+    # Get the total participant teams in a challenge phase
+    participant_team_count = submissions.values_list('participant_team', flat=True).distinct().count()
+
+    # Get the Submission count for Submitted/Running/Failed/Cancelled/Finished/Submitting Submissions
+    submitted_count = submissions.filter(status='submitted').count()
+
+    running_count = submissions.filter(status='running').count()
+
+    failed_count = submissions.filter(status='failed').count()
+
+    cancelled_count = submissions.filter(status='cancelled').count()
+
+    finished_count = submissions.filter(status='finished').count()
+
+    submitting_count = submissions.filter(status='submitting').count()
+
+    submission_status_counts = {'submitted': submitted_count,
```

Comment:
Add more fields like:
1. `flagged_count` which you can get using `submissions.filter(is_flagged=True)`
2. `public_submissions_count` which you can get using `submissions.filter(is_public=True)`